css-layouts/flexbox/src/exercises/exercise2/index.js

```diff
@@ -0,0 +1,136 @@
+/*
+  Next, we're going to apply what we've learned about base-12 grids
+  to arrange some content blocks. 
+
+  The Block prop accepts either an integer representing a single width,
+  or an object representing the widths specified at certain breakpoints.
+
+  The breakpoints provided in the compoment are "all", "small", "medium",
+  and "large". These are intended to match up to mobile devices, medium-range
+  devices like tablets, and then desktops. 
+  
+  In practice, even mobile devices have a large range of pixel sizes
+  so you will see many more breakpoints values in between 
+  the ones set here. 
+
+  For example, if one were to pass in the following size object
+  to the Block component:
+
+  ```
+  size={{
+    small: 12
+    medium: 6
+    large: 4
```

Comment:
Would be great to talk about how a breakpoint for `medium` also applies to `large` if no `large` option is set!